fe/fe-core/src/test/java/com/starrocks/analysis/CreateMaterializedViewTest.java

```diff
@@ -1524,27 +1525,165 @@ public void testAsHasStar() {
                 ") " +
                 "as select k1 ss, *  from tbl1;";
         try {
-            UtFrameUtils.parseStmtWithNewParser(sql, connectContext);
+            StatementBase statementBase = UtFrameUtils.parseStmtWithNewParser(sql, connectContext);
```

Comment:
do you has the case to test two table join with the same column name each.
select a.*, b.* from a join b
and a and b both have column col1

Reply:
add `testAsHasStarWithSameColumns` and `testMVWithSameColumns`